src/buildercore/core.py

```diff
@@ -331,10 +331,8 @@ def _ec2_connection_params(stackname, username, **kwargs):
 
 @contextmanager
 def stack_conn(stackname, username=config.DEPLOY_USER, node=None, **kwargs):
-    if 'user' in kwargs:
-        LOG.warning("found key 'user' in given kwargs - did you mean 'username' ??")
-
-    data = stack_data(stackname, ensure_single_instance=False)
+    ensure('user' not in kwargs, "found key 'user' in given kwargs - did you mean 'username' ??")
+    data = ec2_data(stackname, state='running')
```

Comment:
looks like this defers to `find_ec2_instances` whose `state` parameter is 'running' by default, so no need to redeclare it here

Reply:
... actually, `ec2_data` is `None` by default which is passed to `find_ec2_instances`. 'running' will need to be preserved